src/js/components/Grommet/__tests__/Grommet-test.js

```diff
@@ -85,7 +85,7 @@ describe('Grommet', () => {
 
   test('background', () => {
     const component = renderer.create(
-      <Grommet full background="blue">
+      <Grommet full background="#0000ff">
```

Comment:
why is this needed? for easier comparison with the snapshot result?

Reply:
We are now more accurate setting the text color based on the background color. For example, if you give a background color of "red", we don't know whether to choose light or dark text color, so we leave the text color unset in that case.

I've updated the stories and tests to use hex color values more often, so we can check the mapping more. But, there are still some cases where we leave CSS color names, so we can verify that we don't set a corresponding text color in those cases.